### ISSUE TITLE
Resize Text F69: What is considered "clipped, truncated or obscured"? A letter? A few pixels?

### ISSUE THREAD

**Issue by kimviens** [w3c/wcag#2573]
Hi everyone, I would like to ask about a situation I am unable to determine the answer through the WCAG documentation.

What is considered  "clipped, truncated or obscured" for [F69: Failure of Success Criterion 1.4.4 when resizing visually rendered text up to 200 percent causes the text, image or controls to be clipped, truncated or obscured](https://www.w3.org/WAI/WCAG21/Techniques/failures/F69) to be applicable?

Situation 1: When resizing the text to 200%, the word "related" is missing the letter "d" as the word is clipped. The missing letter "d" is the only clipped text on the whole page. Is this a failure of 1.4.4?
![screenshot of the word related missing the letter d](https://user-images.githubusercontent.com/56558696/183480726-fd6ed63d-b695-4f8d-99ed-ed1070bc3b11.png)

Situation 2: When resizing the text to 200%, the word "jello" is missing part of its bottom half. Is this a failure of 1.4.4?
![screenshot of the word jello missing a bit of the bottom half](https://user-images.githubusercontent.com/56558696/183480452-84b8c8fc-f6e4-4bdd-9ff8-1eed16fd7242.png)

Situation 3: When resizing the text to 200%, nothing else but a few pixels off the word "hello" is missing. Is this a failure of 1.4.4?
![screenshot of the word hello with a few pixels missing, all letters are fully visible](https://user-images.githubusercontent.com/56558696/183481088-1b9751e2-244b-4059-9f8e-373dfc5eefe2.png)

These are edge cases, but surprisingly these cases happen more frequently than expected. I would like to know if there can be more specificities over what constitutes "clipped, truncated or obscured".

**Comment by patrickhlauke** [comment]
i would say technically these are all failures, though the last example is a very mild case (but since WCAG is binary, and doesn't have levels of severity, it's still a hard fail)

**Comment by kimviens** [comment]
@patrickhlauke Thank you, this is exactly what I needed to hear. I am not sure how it works for closing an issue, but my question is answered unless someone has another opinion.

**Comment by bruce-usab** [comment]
FWIW, I would not fail Situation 3, except maybe depending on the rendered font size.  It is subjective, but I am of the opinion that there are enough pixels (figure is closed) so not confuse with a c or something.  By comparison, the o in Jello in Situation 2 is problematic.

No question that your Situations 1 and 2 are examples meant to be addressed by F69.

**Comment by kimviens** [comment]
@bruce-usab, I would agree with your interpretation of situation 3. Maybe the same logic applied to graphics in 1.4.11 Non-Text Content could be applied here? If the clipping, truncation or obscurity can lead to misinterpretations then it is a failure.

Would it be possible to ask for adding these edge cases to F69? This is a common issue that leads to differing reports depending on the auditor's strictness.

I see that there is a note: "The Working Group has discovered many misunderstandings about how to test this failure. We are planning to revise this failure in a future update. Until then, if the content passes the success criterion using any of the listed sufficient techniques, then it does not meet this failure." I think I must have missed this as I am not great at reading.

My problem is that F69 is the technique that is one of the closest techniques to describe "without loss of content or functionality". I think this is a recurring difficulty with WCAG. "Loss of content or functionality" is often used but never defined.

**Comment by bruce-usab** [comment]
There has been reluctance, which I agree with, to include examples in Understanding that are hold-your-nose-and-pass.

**Comment by kimviens** [comment]
Thank you for your reply @patrickhlauke! I am currently attempting to create a testing methodology, this is why I am asking so many questions. My goal is to improve consistency in my reporting. This is driving me to really dig into the WCAG documentation, at the same time, while I am there I tell myself I might as well flag things that I don't understand. In case this can help the documentation too! 😄

**Comment by bruce-usab** [comment]
Thank you @kimviens for taking the time to circle back and close this issue!

**Comment by shawnthompson** [comment]
@alastc I wonder if we can create a label to flag any discussions in issues that would require more guidance added to the Understanding documentation once the new Task force, lead by @mbgower, will be put in place?

**Comment by mbgower** [comment]
@shawnthompson, Alastair, Bruce and I are meeting next week to discuss procedures and methods to get through the Backlog task force. We can certainly discuss appropriate labelling.

@kimviens it doesn't solve the subjectivity problems you're encountering, but I published a verification process several years back that attempts to provide a cohesive method of validating all WCAG requirements. You can look at it here https://www.ibm.com/able/toolkit/verify/overview
Any feedback is welcome. I'm currently working on updating the method to include WCAG 2.2 additions.

**Comment by kimviens** [comment]
@mbgower, you are the one who working on the IBM testing methodologies? I have been referring to [IBM Accessibility Requirements](https://www.ibm.com/able/requirements/requirements/) for a while now. The content from IBM is amazing! Unfortunately, this resource also does not specify what they mean by:

To meet this requirement, confirm that scaling works and ensure there are no visual or functional issues as a result of the increase, such as:
- a lack of appropriate line wrapping
- truncated, overlapped or obscured text
- missing scrollbars when content is zoomed

Since we are now talking about needing more guidance on this issue. I think I will reopen the issue.
